run-kobiton-test.md

```diff
@@ -0,0 +1,171 @@
+    sessionName:        'Automation test session',
+    sessionDescription: '', 
+    deviceOrientation:  'portrait',  
+    captureScreenshots: true, 
+    app:                'kobiton-store:{app_id}', 
+    deviceGroup:        'KOBITON', 
+    deviceName:         {device_name},
+    platformVersion:    {device_version},
+    platformName:       'Android' 
+  }
+  ```
+  >For more information on how to automation test on Kobiton, follow [this tutorial.](https://docs.kobiton.com/automation-testing/using-kobiton-for-automation-testing/)
+
+  6. Push your changes to Github. Travis CI will install the neccessary dependencies and then run the test on Kobiton. 
+  ![](assets/1_build_complete.jpg)
+
+## 3. Secure your Kobiton API Key when triggering a Travis CI build
+There are several methods to attach an enviroment variable to Travis CI.
+In this guide, we will add our Username and API key to `.travis.yml` file. The API Key will be encrypted for security purposes.
+  > To encrypt the variables, we need `Travis` package to be installed. Execute the blow command to install Travis:  
+  > `gem install travis`
+    
+1. In your repository directory, run:  
+    `travis encrypt KOBITON_API_KEY={your_api_key_here} --add env.global`  
+    This will add a secure encrypted key to your `.travis.yml`  
+    Sample result:
+    ```yml
+    env:
+      global:
+        - secure: 'qU+dA2z60akrMEUor4wfjAb/9k9k6HI/Q7xBKpibQsu'
+    ```
+2. Add your Kobiton username to the yml file.
+  
+    ```yml
+    env:
+      global:
+        - KOBITON_USERNAME={your_kobiton_username}
+        - secure: 'qU+dA2z60akrMEUor4wfjAb/9k9k6HI/Q7xBKpibQsu'
+    ```
+
+The final outcome of `.travis.yml` file should look like this:
+  ```yml
+  language: node_js
+  node_js:
+    - '7'
+
+  env:  
+    global:
+      - KOBITON_USERNAME={your_kobiton_username}
+      - secure: 'qU+dA2z60akrMEUor4wfjAb/9k9k6HI/Q7xBKpibQsu'
+
+  script: npm run android-app-test
+  ```
+  
+  >For other methods to secure your enviromental variables, visit [here.](https://docs.travis-ci.com/user/environment-variables/)
+
+3. Open your test script, replace your Kobiton username and API key with enviromental variable name in `kobitonServerConfig`.
+    ```javascript
+    var kobitonServerConfig = {
+      protocol: 'https',
+      host: 'api.kobiton.com',
+      auth: `${process.env.KOBITON_USERNAME}:${process.env.KOBITON_API_KEY}`
+    }
+    ```
+
+4. Push your changes to Github to verify the build in Travis CI.  
+5. Travis CI will export your environment variables from `.travis.yml`. The encrypted key will be showed as `[secured]`.
+  
+![](assets/2_travis_env.jpg)  
+
+## 4. Get session information through REST API
+Kobiton already provides a Node.js sample on how to get session information using Kobiton REST API. 
+  - Go to https://github.com/kobiton/samples
+  - Go to the `REST API` folder and follow the instruction in `README.md` file.
+
+## 5. (Optional) Use cache to improve Travis CI building speed
```

Comment:
I'm not sure if this part is necessary. Please let me know if it's needed for this guideline.